<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Parameter_DigitalPin_Form</class>
 <widget class="QWidget" name="Parameter_DigitalPin_Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1025</width>
    <height>675</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1025</width>
    <height>675</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#Parameter_DigitalPin_Form
{
	border-image: url(:/resource/images/common/Transparent_Black_70.png);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>114</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>259</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="widgetBg" native="true">
     <property name="minimumSize">
      <size>
       <width>471</width>
       <height>480</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>471</width>
       <height>480</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <widget class="PushButtonDone" name="pushButton">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>390</y>
        <width>112</width>
        <height>82</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Nexa Light</family>
        <pointsize>26</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	border-image: url(:/resource/images/upload_setting/Done.png);
	border-color: rgba(255, 255, 255, 0);
}
QPushButton:hover
{
	border-image: url(:/resource/images/upload_setting/Done_OVER.png);
	border-color: rgba(255, 255, 255, 0);
}</string>
      </property>
      <property name="text">
       <string>Done</string>
      </property>
      <property name="shortcut">
       <string>Ctrl+Return</string>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>420</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 17pt &quot;Microsoft YaHei UI&quot;;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>420</y>
        <width>31</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(105, 105, 105);
font: 13pt &quot;Microsoft JhengHei&quot;;</string>
      </property>
      <property name="text">
       <string>Pin:</string>
      </property>
     </widget>
     <widget class="QLabel" name="backgourndimage">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>421</width>
        <height>381</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>resource/images/parameter_input/DigitalPin/Arduino Starter.png</pixmap>
      </property>
     </widget>
     <widget class="QPushButton" name="DigitalPin_0">
      <property name="geometry">
       <rect>
        <x>403</x>
        <y>13</y>
        <width>16</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	border-color: rgba(0, 255, 0, 0);
}

QPushButton:pressed
{
	border-color: rgba(255, 0, 0, 0);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="src.qrc">
        <normaloff>:/resource/images/parameter_input/AnalogPin/SelectBox.png</normaloff>:/resource/images/parameter_input/AnalogPin/SelectBox.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>0</width>
        <height>37</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="DigitalPin_1">
      <property name="geometry">
       <rect>
        <x>388</x>
        <y>13</y>
        <width>16</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="src.qrc">
        <normaloff>:/resource/images/parameter_input/AnalogPin/SelectBox.png</normaloff>:/resource/images/parameter_input/AnalogPin/SelectBox.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>0</width>
        <height>37</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="DigitalPin_2">
      <property name="geometry">
       <rect>
        <x>373</x>
        <y>13</y>
        <width>16</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="src.qrc">
        <normaloff>:/resource/images/parameter_input/AnalogPin/SelectBox.png</normaloff>:/resource/images/parameter_input/AnalogPin/SelectBox.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>0</width>
        <height>37</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="DigitalPin_3">
      <property name="geometry">
       <rect>
        <x>357</x>
        <y>13</y>
        <width>16</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="src.qrc">
        <normaloff>:/resource/images/parameter_input/AnalogPin/SelectBox.png</normaloff>:/resource/images/parameter_input/AnalogPin/SelectBox.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>0</width>
        <height>37</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="DigitalPin_4">
      <property name="geometry">
       <rect>
        <x>342</x>
        <y>13</y>
        <width>16</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="src.qrc">
        <normaloff>:/resource/images/parameter_input/AnalogPin/SelectBox.png</normaloff>:/resource/images/parameter_input/AnalogPin/SelectBox.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>0</width>
        <height>37</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="DigitalPin_5">
      <property name="geometry">
       <rect>
        <x>327</x>
        <y>13</y>
        <width>16</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="src.qrc">
        <normaloff>:/resource/images/parameter_input/AnalogPin/SelectBox.png</normaloff>:/resource/images/parameter_input/AnalogPin/SelectBox.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>0</width>
        <height>37</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="DigitalPin_6">
      <property name="geometry">
       <rect>
        <x>312</x>
        <y>13</y>
        <width>16</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="src.qrc">
        <normaloff>:/resource/images/parameter_input/AnalogPin/SelectBox.png</normaloff>:/resource/images/parameter_input/AnalogPin/SelectBox.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>0</width>
        <height>37</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="DigitalPin_7">
      <property name="geometry">
       <rect>
        <x>297</x>
        <y>13</y>
        <width>16</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="src.qrc">
        <normaloff>:/resource/images/parameter_input/AnalogPin/SelectBox.png</normaloff>:/resource/images/parameter_input/AnalogPin/SelectBox.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>0</width>
        <height>37</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="DigitalPin_8">
      <property name="geometry">
       <rect>
        <x>267</x>
        <y>13</y>
        <width>16</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="src.qrc">
        <normaloff>:/resource/images/parameter_input/AnalogPin/SelectBox.png</normaloff>:/resource/images/parameter_input/AnalogPin/SelectBox.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>0</width>
        <height>37</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="DigitalPin_9">
      <property name="geometry">
       <rect>
        <x>252</x>
        <y>13</y>
        <width>16</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="src.qrc">
        <normaloff>:/resource/images/parameter_input/AnalogPin/SelectBox.png</normaloff>:/resource/images/parameter_input/AnalogPin/SelectBox.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>0</width>
        <height>37</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="DigitalPin_10">
      <property name="geometry">
       <rect>
        <x>237</x>
        <y>13</y>
        <width>16</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="src.qrc">
        <normaloff>:/resource/images/parameter_input/AnalogPin/SelectBox.png</normaloff>:/resource/images/parameter_input/AnalogPin/SelectBox.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>0</width>
        <height>37</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="DigitalPin_11">
      <property name="geometry">
       <rect>
        <x>222</x>
        <y>13</y>
        <width>16</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="src.qrc">
        <normaloff>:/resource/images/parameter_input/AnalogPin/SelectBox.png</normaloff>:/resource/images/parameter_input/AnalogPin/SelectBox.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>0</width>
        <height>37</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="DigitalPin_12">
      <property name="geometry">
       <rect>
        <x>207</x>
        <y>13</y>
        <width>16</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="src.qrc">
        <normaloff>:/resource/images/parameter_input/AnalogPin/SelectBox.png</normaloff>:/resource/images/parameter_input/AnalogPin/SelectBox.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>0</width>
        <height>37</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="DigitalPin_13">
      <property name="geometry">
       <rect>
        <x>192</x>
        <y>13</y>
        <width>16</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="src.qrc">
        <normaloff>:/resource/images/parameter_input/AnalogPin/SelectBox.png</normaloff>:/resource/images/parameter_input/AnalogPin/SelectBox.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>0</width>
        <height>37</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>259</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>114</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PushButtonDone</class>
   <extends>QPushButton</extends>
   <header>PushButtonDone.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
